docs/src/languages/php/_index.md

```diff
@@ -304,16 +210,197 @@ markdownify=false
 
 {{< /codetabs >}}
 
-
-
 {{% config-reader %}}
 [`platformsh/config-reader` Composer library](https://github.com/platformsh/config-reader-php)
 {{% /config-reader%}}
 
-## Runtime configuration
+## PHP settings
+
+You can configure your PHP-FPM runtime configuration by specifying the [runtime in your app configuration](../../create-apps/app-reference.md#runtime).
+
+You can set the [PHP runtime timezone](../../create-apps/timezone.md).
+
+Some commonly used settings are:
+
+| Name | Default | Description |
+|------|-------|-------------|
+| `max_execution_time` | `0` | This is the maximum execution time, in seconds, for your PHP scripts and applications. A value of `0` means there are no time limits. |
+| `max_file_uploads` | `20` | This is the maximum number of files that can be uploaded per request. |
+| `max_input_time` | `-1` | This is the maximum time, in seconds, that your script is allowed to receive input (e.g for file uploads). A value of `-1` means there are no time limits. |
+| `max_input_vars` | `1000` | This is the maximum amount of input variables that are accepted per request. |
+| `memory_limit` | `512M` | The memory limit, in megabytes, that's set for PHP. |
+| `post_max_size` | `8M` | This is the maximum size, in megabytes, per uploaded file. Increase this value to upload larger files. |
+| `zend.assertions` | `-1` | Assertions are optimized and have no impact at runtime. Set assertions to `1` for your local development system. [See more on assertions](https://www.php.net/manual/en/regexp.reference.assertions). |
+| `opcache.memory_consumption` | `64` | This is the number of megabytes available for [the OPcache](./tuning.md#opcache-preloading). Increase this value for large applications with many files.|
+| `opcache.validate_timestamps` | `On` | [The OPcache](./tuning.md#opcache-preloading) checks for updated files on disk. This is necessary to support applications that generate compiled PHP code from user configuration. If you are certain your application doesn't do so then you can disable this setting for a small performance boost. |
```

Comment:
:clipboard: 

```suggestion
| `opcache.validate_timestamps` | `On` | [OPcache](./tuning.md#opcache-preloading) checks for updated files on disk to support apps that generate compiled PHP code from user configuration. If your app doesn't generate such code, you can disable this setting for a small performance boost. |
```

Reply:
I realized this whole section is better explained in the section dedicated to it (./tuning.md#disable-opcache-timestamp-validation). So most of this explanation should be removed in favor of a link. And then than section could link here, if necessary⎄